spyder/plugins/ipythonconsole/utils/manager.py

```diff
@@ -15,7 +15,15 @@
 # Third party imports
 from jupyter_client.utils import run_sync
 import psutil
+from qtconsole.client import QtKernelClient, QtZMQSocketChannel
 from qtconsole.manager import QtKernelManager
+from traitlets import Type, DottedObjectName
+
+
+class SpyderKernelClient(QtKernelClient):
+    # Enable recieving messages on control channel
+    # Useful for pdb completion
+    control_channel_class = Type(QtZMQSocketChannel)

```

Comment:
Please move this class to `spyder/plugins/ipythonconsole/utils/client.py`. Even if it's really small, I think it's useful to have separate files for our subclasses of `QtKernelClient` and `QtKernelManager`.